FWCore/MessageLogger/src/ELmap.cc

```diff
@@ -134,10 +134,12 @@ namespace edm {
   // Global Dump methods (useful for debugging)
   // ----------------------------------------------------------------------
 
+#include <memory>
 #include <sstream>
 #include <string.h>
+#include <vector>
 
-  boost::shared_array<char> ELmapDump(ELmap_limits m) {
+  std::shared_ptr<std::vector<char>> ELmapDump(ELmap_limits m) {
```

Comment:
This function is not used anywhere in CMSSW, and @Dr15Jones confirmed privately that it has never been used in CMSSW. Could you just remove the whole
```cpp
#ifdef ELmapDumpTRACE
...
#endif
```
block?

Reply:
Sure